### ISSUE TITLE
PR Updated not triggering a Job

### ISSUE THREAD

**Issue by georgealton** [tomasbjerre/pull-request-notifier-for-bitbucket#124]
Hi 

we're running bitbuck 4.6.2 and prnfb 2.22. I'm experiencing an issue where when pull request is opened a job is created in jenkins, however when new commits are made against the pr branch a new job isn't. 

below is my trigger configuration ; 

![image](https://cloud.githubusercontent.com/assets/1202151/16276637/2979280e-38a7-11e6-9e13-dc3223c327e1.png)

is there some misconfiguration somewhere?

**Comment by tiberiomoliveira** [comment]
I have the same issues as you, but my Bitbucket is v4.3.2 and the prnfb v2.24. Unfortunately, I have no solution for that. I've even tried to create a button for trigger the job but when I enter into the PR page the button appears for a short time and then it disappears.

**Comment by tomasbjerre** [comment]
The configuration looks good! Should work. And there is nothing in your logs?

"and then it disappears"
When you reload the page? Its suddenly not there?

**Comment by tiberiomoliveira** [comment]
Yes, this is it. When the page finishes to reload the button disappears from the page.

**Comment by tomasbjerre** [comment]
Perhaps you are also using some other plugin?

**Comment by tomasbjerre** [comment]
Does this happen for any user? Did you upgrade from a version where it did work?

**Comment by georgealton** [comment]
we were originally on 4.2 where I had it working - however a couple of weeks ago we went to 4.6 - I'll try and get access to the logs tomorrow as I don't administer the server.

**Comment by tiberiomoliveira** [comment]
No, I don't use any other plugin for now. I only have the prnfb plugin installed on my BitBucket Server.

**Comment by tomasbjerre** [comment]
You should open another issue for your problem with the button. And supply your configuration.

**Comment by tomasbjerre** [comment]
Is the pull request within a repo or from a fork?

**Comment by georgealton** [comment]
It's within a repo, 

here is me opening a PR for a branch to merge to master; 
![image](https://cloud.githubusercontent.com/assets/1202151/16297582/5708eef8-3929-11e6-9422-08063d511bdb.png)

and here is that build triggered in jenkins
![image](https://cloud.githubusercontent.com/assets/1202151/16297616/87b4db3e-3929-11e6-93a3-089d154666a9.png)

i then add a commit to the branch that the PR is for 
![image](https://cloud.githubusercontent.com/assets/1202151/16297896/b19e396c-392a-11e6-8181-ec5aca4c4d17.png)

however there is no job triggered in jenkins

![image](https://cloud.githubusercontent.com/assets/1202151/16297948/e326867e-392a-11e6-80b8-1946d7a6bdb2.png)

**Comment by tomasbjerre** [comment]
And the pull request is within project DPLY and repo saltstack?

**Comment by tomasbjerre** [comment]
What happens if you also check RESCOPED_TO?
You may try to invoke some other webserver and watch the logs of that. To see what the invokation looks like. And if it happens...

**Comment by georgealton** [comment]
yea project DPLY and repo saltstack 

just enabled rescoped_to and made a commit on the source branch 

![image](https://cloud.githubusercontent.com/assets/1202151/16300213/a58946b0-3936-11e6-9753-5f5c6f6172c4.png)

i'll try and make a http server to act as an endpoint for receiving the notifications instead of jenkins

**Comment by georgealton** [comment]
ok so it seems im not getting the request on a different server either ; 

declining and reopening triggers a request
<img width="1411" alt="reopened" src="https://cloud.githubusercontent.com/assets/1202151/16300877/24f1ac5a-393a-11e6-8c8d-f2c3552cd0bb.png">

committing doesn't send a new one
<img width="1419" alt="with_new_commit" src="https://cloud.githubusercontent.com/assets/1202151/16300880/274613ba-393a-11e6-9b20-cef6f2f8c518.png">

**Comment by tomasbjerre** [comment]
Perhaps a permission issue... and the logs of Bitbucket Server shows no hint?
What if you make a commit as an admin user?

**Comment by georgealton** [comment]
my user has admin level privileges (not system admin)

currently trying to get access to the log directory on the server hosting bitbucket...

**Comment by tomasbjerre** [comment]
I may have recreated this bug locally. Am seeing this in my log:
```
[INFO] [talledLocalContainer] 2016-06-23 16:07:10,877 ERROR [threadpool:thread-8]  c.a.s.i.c.StateTransferringExecutor Error while processing asynchronous task
[INFO] [talledLocalContainer] java.lang.NullPointerException: null
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.service.PrnfbVariable$40.resolve(PrnfbVariable.java:435) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.service.PrnfbVariable.resolve(PrnfbVariable.java:568) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.service.PrnfbRenderer.getRenderedString(PrnfbRenderer.java:68) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.service.PrnfbRenderer.renderVariable(PrnfbRenderer.java:79) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.service.PrnfbRenderer.render(PrnfbRenderer.java:56) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.listener.PrnfbPullRequestEventListener.notify(PrnfbPullRequestEventListener.java:142) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.listener.PrnfbPullRequestEventListener.handleEvent(PrnfbPullRequestEventListener.java:239) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.listener.PrnfbPullRequestEventListener.access$000(PrnfbPullRequestEventListener.java:54) ~[na:na]
[INFO] [talledLocalContainer] 	at se.bjurr.prnfb.listener.PrnfbPullRequestEventListener$1.run(PrnfbPullRequestEventListener.java:83) ~[na:na]
[INFO] [talledLocalContainer] 	at com.atlassian.stash.internal.concurrent.StateTransferringExecutor$StateTransferringRunnable.run(StateTransferringExecutor.java:70) ~[StateTransferringExecutor$StateTransferringRunnable.class:na]
[INFO] [talledLocalContainer] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_91]
[INFO] [talledLocalContainer] 	... 1 frame trimmed
```

**Comment by tomasbjerre** [comment]
The commit notification issue should be fixed now in 2.26.

**Comment by georgealton** [comment]
thanks @tomasbjerre !

working for me on 2.26 
![image](https://cloud.githubusercontent.com/assets/1202151/16333808/43dca0f0-39f5-11e6-9248-b4ad902ec132.png)
